### DISCUSSION TITLE
New version has home tab open every time the program is opened?

### DISCUSSION THREAD

**Discussion by Kurineko** [sumatrapdfreader/sumatrapdf#4056] [Q&A]
Hey, just wanted to know if there was a way to disable this function from the newer version of the program, where the home tab opens in the background automatically when opening any pdf file, even after closing it as a tab and then closing the program as a whole, it sort of stumps my workflow for keeping track of dates that are on the pdf files I open
here is a video in case I'm not explaining myself well:
https://github.com/sumatrapdfreader/sumatrapdf/assets/36141696/c5d66e04-d79e-4bd8-a185-29c6b4988714

**Comment by GitHubRulesOK** [comment]
advanced settings `NoHomeTab = true`

**Comment by Kurineko** [comment]
Omg thanks so much!
